### PR TITLE
Add nanopb 0.3.9.1

### PR DESCRIPTION
**Pull request by rsgowman**
Also update the version scheme for the podfile, since 4 version
components aren't compatible with semver.

### CODE REVIEW

File: nanopb/0.3.9.1/nanopb.podspec [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Pod::Spec.new do |s|
+  s.name         = "nanopb"
+  s.version      = "0.3.901"
+  s.summary      = "Protocol buffers with small code size."
+
+  s.description  = <<-DESC
+                    Nanopb is a small code-size Protocol Buffers implementation
+                    in ansi C. It is especially suitable for use in
```
Comment (paulb777) [inline nanopb/0.3.9.1/nanopb.podspec]: nit: ANSI C

Comment (rsgowman) [reply nanopb/0.3.9.1/nanopb.podspec]: It's verbatim from the nanopb site; I'd prefer to leave it as is. (But am fine with fixing it too; let me know if you'd like that to happen.)

Comment (paulb777) [reply nanopb/0.3.9.1/nanopb.podspec]: OK - I'm fine to leave it then.

**Comment by paulb777** [comment]
--------------------------------------------------------------------------------
 🎉  Congrats

 🚀  nanopb (0.3.901) successfully published
 📅  October 24th, 13:49
 🌎  https://cocoapods.org/pods/nanopb
 👍  Tell your friends!
--------------------------------------------------------------------------------

**Comment by paulb777** [comment]
I needed to add --allow-warnings since the version doesn't match the tag.

And I had a simulator issue that required the workaround at https://github.com/CocoaPods/CocoaPods/issues/7454

File: nanopb/0.3.9.1/nanopb.podspec [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                   DESC
+
+  s.homepage     = "https://github.com/nanopb/nanopb"
+  s.license      = { :type => 'zlib', :file => 'LICENSE.txt' }
+  s.author       = { "Petteri Aimonen" => "jpa@nanopb.mail.kapsi.fi" }
+  s.source       = { :http => "https://github.com/nanopb/nanopb/archive/#{s.version}.tar.gz" }
+  s.source       = { :git => "https://github.com/nanopb/nanopb.git", :tag => "0.3.9.1" }
+
+  s.requires_arc = false
+  s.xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) PB_FIELD_32BIT=1 PB_NO_PACKED_STRUCTS=1 PB_ENABLE_MALLOC=1' }
```
Comment (dnkoutso) [inline nanopb/0.3.9.1/nanopb.podspec]: Is changing the users xcconfig versus the pod target xcconfig (`s.pod_target_xcconfig`) intentional?

**Pull request merged**